swatch-core/src/main/java/org/candlepin/subscriptions/db/model/EventRecord.java

```diff
@@ -74,24 +77,32 @@ public EventRecord(Event event) {
     this.timestamp = event.getTimestamp();
   }
 
-  @Id private UUID id;
+  @Column(name = "event_id")
+  private UUID eventId;
 
   @Column(name = "account_number")
   private String accountNumber;
 
+  @Id
   @Column(name = "org_id")
   private String orgId;
 
+  @Id
   @Column(name = "event_type")
   private String eventType;
 
+  @Id
   @Column(name = "event_source")
   private String eventSource;
 
+  @Id
   @Column(name = "instance_id")
   private String instanceId;
 
-  private OffsetDateTime timestamp;
+  @Column(name = "span_id")
+  private String spanId;
```

Comment:
This should be a UUID type instead for performance reasons.

Reply:
Also thinking... maybe this should be called `meteringBatchId` or something similar. Otherwise, in the case that we are using tracing, we would end up with two spanIds, one generated by instrumentation and this one.

Reply:
Changed to UUID type and the name to `meteringBatchId`.